Makefile

```
@@ -1,4 +1,5 @@
 VERSION = $(shell git describe --tags --dirty --always)
+ZED_VERSION := $(shell go list -f '{{.Version}}' -m github.com/brimdata/zed)
 LDFLAGS = -s -X github.com/brimdata/brimcap/cli.Version=$(VERSION)
```

Nits: No need for quotes. And don't separate `VERSION` from `LD_FLAGS`.
```suggestion
LDFLAGS = -s -X github.com/brimdata/brimcap/cli.Version=$(VERSION)
ZED_VERSION := $(shell go list -f {{.Version}} -m github.com/brimdata/zed)
```